src/wiki/templates/wiki/article.html

```
@@ -43,6 +44,95 @@ <h1 id="article-title">{{ urlpath.item_type|title }}: {{ article.current_revisio
 </div>
 {% endblock tags %}
 
+{% block comments %}
+<h3>{% trans 'Comments' %}</h3>
+{% get_comment_list for urlpath as comment_list %}
+<div class="comments">
+    {% for comment in comment_list|fill_tree|annotate_tree %}
+        {% if comment.open %}
+            <ul>
+        {% else %}
+            </li>
+        {% endif %}
+        <li class="comment_li" id="c{{ comment.id }}">
+            <div class="comment">
+                <div class="comment_info">
+                    <div class="comment_user">{{ comment.user.get_full_name }}</div>
+                    <div class="comment_data">
+                        {{ comment.submit_date|date:"d.m.Y, H:i" }}
+                        <button type="button" class="js-reply-button" data-id="{{ comment.id }}">Reply</button>
+                    </div>
+                </div>
+                <div class="comment_text">
+                    {{ comment.comment }}
+                </div>
+                {% if user.is_authenticated %}
+                {% get_comment_form for urlpath as form with comment.id %}
+                <div id="form-comment">
+                    <form action="/comments/post/" method="post" id="js-reply-form-{{ comment.id }}" data-id="{{ comment.id }}" hidden>{% csrf_token %}
+                     {{ form.comment }}
+                     {{ form.content_type }}
+                     {{ form.object_pk }}
+                     {{ form.timestamp }}
+                     {{ form.parent }}
+                     {{ form.security_hash }}
+                        <p>
+                            <input type="text" name="honeypot" class="honeypot">
```

honeypot можно было спрятать не css a type="hidden"

---

Из доки:
The comment form includes a “honeypot” field. It’s a trap: if any data is entered in that field, the comment will be considered spam (spammers often automatically fill in all fields in an attempt to make valid submissions).
The default form hides this field with a piece of CSS and further labels it with a warning field; if you use the comment form with a custom template you should be sure to do the same.